CalibTracker/SiStripLorentzAngle/src/LA_Filler.cc

```diff
@@ -85,11 +86,14 @@ granularity(const SiStripDetId detid, const float tthetaL, const Long64_t TFE_in
 }
 
 std::string LA_Filler_Fitter::
-subdetLabel(const SiStripDetId detid) { return detid.subDetector()==SiStripDetId::TOB? "TOB" : "TIB";}
+subdetLabel(const SiStripDetId detid) { return detid.subDetector()==SiStripDetId::TOB ? "TOB" : "TIB";}
 std::string LA_Filler_Fitter::
 moduleLabel(const SiStripDetId detid) { return subdetLabel(detid) + "_module"+boost::lexical_cast<std::string>(detid());}
 std::string LA_Filler_Fitter::
-layerLabel(const SiStripDetId detid) {
-  unsigned layer = detid.subDetector() == SiStripDetId::TOB ? TOBDetId(detid()).layer() : TIBDetId(detid()).layer();
-  return subdetLabel(detid)+"_layer"+boost::lexical_cast<std::string>(layer)+(detid.stereo()?"s":"a");
+layerLabel(const SiStripDetId detid) const {
+  const bool isTIB = detid.subdetId() == StripSubdetector::TIB;
+  unsigned layer = isTIB ? tTopo_->tibLayer(detid) : tTopo_->tobLayer(detid);
+  bool stereo = isTIB ? tTopo_->tibStereo(detid) : tTopo_->tobStereo(detid);
+  
+  return subdetLabel(detid)+"_layer"+boost::lexical_cast<std::string>(layer)+(stereo?"s":"a");
```

Comment:
in light of  #19645 I guess you can get rid of the ugly `boost::lexical_cast` in favor of `std::to_string`